src/engine/renderer/gl_shader.cpp

```diff
@@ -417,84 +466,26 @@ static void AddConst( std::string& str, const std::string& name, float v1, float
 	str += Str::Format("const vec2 %s = vec2(%.8e, %.8e);\n", name, v1, v2);
 }
 
-static std::string GenVersionDeclaration() {
-	// Basic version declaration
+inline std::string GenVersionDeclaration( const std::vector<addedExtension_t> &addedExtensions ) {
```

Comment:
It never makes sense to use `inline` in a .cpp file. Use `static`

Reply:
Fixed. This was a leftover from when I tried various things to see if the compiler would accept to use iterators on C arrays passed as argument (spoiler: it didn't work 🥲️).